### DISCUSSION TITLE
New pane type (e.g. "monitor")

### DISCUSSION THREAD

**Discussion by globalshrug** [tmux/tmux#4131] [General]
I was looking for this as a feature or plugin but didn't find it. Basically, I often have a "monitoring" pane open with e.g. a log file or something, then I have two or more other panes where I'm working.

When I e.g. `<C-b> o`, I have to go _through_ my monitoring pane to get to either of my working panes. There are crude work-arounds for this e.g. re-arranging panes and using explicit keys/numbers corresponding to specific panes. But it would be a great feature to be able to set a specific pane into a different mode (e.g. "monitor" or "deactivated", ...), and then have regular pane actions "skip over" that particular pane. So in deactivated mode, pane #2 would be skipped over when hitting `<C-b> o`, allowing it to function purely as a visual/monitoring pane. A tmux command would be issued to re-set that pane to a normal type.

I'm curious if this idea appeals to anyone else.

**Comment by nicm** [comment]
You can do this fairly easily by setting a user option on your special panes and then rebinding `C-b o` to check for the option and skip those panes. I think you should also be able to remove from `choose-tree` by using `-f`.
